gp-media-library/gpml-apc-attach-files-to-post.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Gravity Wiz // Gravity Forms // Attach Files uploaded via GPML to Posts created with GFAPC + ACF
+ * 
+ * Attach Files uploaded via GPML to Posts created with GFAPC + ACF
+ *
+ * Instructions:
+ *  1. Install per https://gravitywiz.com/how-do-i-install-a-snippet/
+ */
+add_action( 'gform_advancedpostcreation_post_after_creation', function( $post_id, $feed, $entry, $form ) {
+	foreach ( $feed['meta']['postMetaFields'] as $post_meta_field ) {
+		$field_id         = rgar( $post_meta_field, 'value' );
+		$field            = GFAPI::get_field( $form, $field_id );
+		$is_media_library = rgar( $field, 'uploadMediaLibrary' );
+
+		if ( $is_media_library ) {
+			$file_ids = gp_media_library()->get_file_ids( $entry['id'], $field->id );
+			foreach ( $file_ids as $file_id ) {
+				wp_update_post( array(
+					'ID'          => $file_id,
+					'post_parent' => $post_id,
+				) );
+			}
+			set_post_thumbnail( $post_id, $file_id );
```

Comment:
This would use the last `$file_id` as the featured image.

I think for `set_post_thumbnail`, we'd need to find the field that it mapped to it a la https://github.com/gravitywiz/gp-media-library/blob/master/class-gp-media-library.php#L672

Reply:
There is no need to set the post thumbnail. The user only wants to attach the image to the post which is handled by the function above.

Reply:
@claygriffiths @spivurno darn! I added it by mistake. I know we don't need it, didn't use it for testing anyways. Will update today